src/poliastro/_math/linalg.py

```diff
@@ -0,0 +1,7 @@
+import numpy as np
+from numba import njit as jit
+
+
+@jit
+def norm_fast(arr):
```

Comment:
Can we perhaps rename this to just `norm`, and import it as such? That way we minimize the lines changed too.

```suggestion
def norm(arr):
```

Reply:
(And, in modules where we have units-compatible `norm` and this function, we can do `from poliastro._math.linalg import norm as norm_fast`)

Reply:
Yeah, this seems to be better!